.github/workflows/test.yml

```diff
@@ -36,7 +36,17 @@ jobs:
       - name: Run build
         run: npm run build
 
-      - name: Run all tests
+      - name: Run all tests (without coverage)
```

Comment:
I opted to make this an entirely duplicated "job" within the workflow just to be explicit at what was going on / make it easier to see that tests are intentionally being run twice.

That being said, this choice results in duplicated env settings so I wanted to call it out.

I'd be happy to change this to a single job that runs both commands via `&`, and therefore avoids duplication of the env settings.

Reply:
I think it's worth it to have the clean separation. Only a handful of lines are duplicated and we can remove this when the [upstream bug](https://github.com/kulshekhar/ts-jest/issues/4193) is fixed.